.github/workflows/publish.yml

```diff
@@ -1,46 +0,0 @@
-name: publish
```

Comment:
Make sure to add the publish workflow back once you're ready (only changes to the CI since its a move from parcel -> sveltekit is to change `dist` to `build`.